relay-general/src/store/normalize.rs

```diff
@@ -663,6 +663,7 @@ pub struct LightNormalizationConfig<'a> {
     pub measurements_config: Option<&'a MeasurementsConfig>,
     pub breakdowns_config: Option<&'a BreakdownsConfig>,
     pub normalize_user_agent: Option<bool>,
+    pub normalize_transaction_name: Option<bool>,
```

Comment:
nit: Does this need to be an `Option<bool>`? I don't know why `normalize_user_agent` is one to be honest, I feel like a simple `bool` would do just fine.